Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/UserController.cs

```diff
@@ -177,8 +178,24 @@ public IActionResult GetRole()
                 }
                 return Ok(false);
             }
+            catch (InvalidApiRequestException apiEx)
```

Comment:
Exceptions.InvalidApiRequestException: Bad Request Check `Errors` for a list of errors returned by the API.
at SendGrid.ErrorChecker.CheckForErrors(System.Net.Http.HttpResponseMessage response, System.IO.Stream stream) at offset 182
at SendGrid.ErrorChecker.<CheckForErrorsAsync>d__1.MoveNext() at offset 148
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(System.Threading.Tasks.Task task) at offset 38
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task task) at offset 40
at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
at SendGrid.Web.<DeliverAsync>d__7.MoveNext() at offset 332

Because InvalidApiRequestException is the Exception that has occurred